spring-cloud-aws-autoconfigure/src/main/java/io/awspring/cloud/autoconfigure/messaging/SnsAutoConfiguration.java

```diff
@@ -81,16 +85,28 @@
 		return clientFactoryBean;
 	}
 
+	@ConditionalOnMissingAmazonClient(SnsMessageManager.class)
+	@Bean
+	public SnsMessageManager snsMessageManager(SnsProperties snsProperties) {
+		// when used with localstack what should be verification region?
+		if (snsProperties.getEndpoint() != null || regionProvider == null) {
```

Comment:
If properties are using localstack and have URL local not sure which region should be selected for verification.